modules/ipi-install-network-requirements.adoc

```diff
@@ -26,25 +28,166 @@ When using a VLAN, each NIC must be on a separate VLAN corresponding to the appr
 ====
 
 [discrete]
-== Configuring the DNS server
+== DNS requirements
 
 Clients access the {product-title} cluster nodes over the `baremetal` network. A network administrator must configure a subdomain or subzone where the canonical name extension is the cluster name.
 
+[source,text]
 ----
 <cluster_name>.<domain>
 ----
 
 For example:
 
+[source,text]
 ----
 test-cluster.example.com
 ----
 
 {product-title} includes functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses. After the nodes are registered with the API, the cluster can disperse node information without using CoreDNS-mDNS. This eliminates the network traffic associated with multicast DNS.
 
+In {product-title} deployments, DNS name resolution is required for the following components:
+
+* The Kubernetes API
+* The {product-title} application wildcard ingress API
+
+Reverse DNS resolution is also required for the Kubernetes API, the provisioner node, the control plane nodes, and the worker nodes.
+
+A/AAAA records are used for name resolution and PTR records are used for reverse name resolution. {op-system-first} uses the reverse records or DHCP to set the hostnames for all the nodes. Additionally, the reverse records are used to generate the certificate signing requests (CSR) that {product-title} needs to operate.
+
+Installer-provisioned installation includes functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses.
+
+The installer creates the following DNS records for an installer-provisioned {product-title} cluster. In each record, `<cluster_name>` is the cluster name and `<base_domain>` is the base domain that you specify in the `install-config.yaml` file. A complete DNS record takes the form: `<component>.<cluster_name>.<base_domain>.`.
+
+.Required DNS records
+[cols="1a,3a,5a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+.2+a|Kubernetes API
+|`api.<cluster_name>.<base_domain>.`
+|An A/AAAA record, and a PTR record, identify the API load balancer. These records must be resolvable by both clients external to the cluster and from all the nodes within the cluster.
+
 [IMPORTANT]
 ====
-You must create a DNS entry for the `api.<cluster_name>.<domain>` domain name on the external DNS because removing CoreDNS causes the local entry to disappear. Failure to create a DNS record for the `api.<cluster_name>.<domain>` domain name in the external DNS server precludes worker nodes from joining the cluster.
+The API server must be able to resolve the worker nodes by the hostnames
```

Comment:
I'm not sure this is true (I've been told the same thing by others, but my experience differs), but even if it is it's not something deployers need to worry about. Node name records are provided by the internal DNS server in IPI deployments. If this is not working it's a bug on our end, not something the customer needs to fix.

Reply:
Is this sentence missing a period?